mbed_lstools/lstools_base.py

```diff
@@ -28,7 +28,7 @@
 class MbedLsToolsBase:
     """ Base class for mbed-lstools, defines mbed-ls tools interface for mbed-enabled devices detection for various hosts
     """
-    def __init__(self):
+    def __init__(self, **kwargs):
```

Comment:
That's fine, it's just harder to pick up on a first-time read of the code. Not that big of an issue though. And also the alternative is what the mbed OS tools do, which is not that great :stuck_out_tongue: 

Just look at this beatuy: https://github.com/ARMmbed/mbed-os/blob/master/tools/build_api.py#L365